packages/server/src/migrations/v13.ts

```diff
@@ -0,0 +1,6 @@
+import { PoolClient } from 'pg';
+
+export async function run(client: PoolClient): Promise<void> {
+  await client.query('ALTER TABLE "ServiceRequest" ALTER COLUMN "orderDetail" DROP NOT NULL');
```

Comment:
What does this do?

Reply:
Most of the search index columns were created in the initial schema.  Note that they did not use the "NOT NULL" modifier:
https://github.com/medplum/medplum/blob/main/packages/server/src/migrations/v1.ts

These 2 columns were created later, and for no particular reason did include "NOT NULL".

This diff fixes a bug where removed properties are indexed by setting them to null.  For that to work, all search columns need to follow the same pattern, and to allow null values.